README.rst

```diff
@@ -516,18 +518,18 @@ only those exact fixers are enabled whether or not level is set.
 With the ``--config-file`` option you can specify the path to the
 ``.php_cs`` file.
 
-By using ``--using-cache`` option you can set if caching
-mechanism should be used.
+By using ``--using-cache`` option with ``yes`` or ``no`` you can set if caching
```

Comment:
I'm confused;
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/README.rst#caching
``` The caching mechanism is enabled by default. ```
Is this on master only? (my PR targets master)